streaming/java/streaming-state/src/main/java/org/ray/streaming/state/keystate/state/OneOutState.java

```diff
@@ -0,0 +1,14 @@
+package org.ray.streaming.state.keystate.state;
+
+/**
+ * one value per state.
+ */
+public interface OneOutState<O> extends State {
```

Comment:
So why is it named `OneOutState` where the noun `Out` is quite confusing.

Reply:
You are right, OneState is more straightforward, Thanks